components/event-publisher-proxy/README.md

```diff
@@ -76,21 +76,21 @@ curl -v -X GET \
-| TOKEN_ENDPOINT          |               | The Authentication Server Endpoint to provide Access Tokens.                                  |
-| EMS_PUBLISH_URL         |               | The Messaging Server Endpoint that accepts publishing CloudEvents to it.                      |
-| BEB_NAMESPACE           |               | The name of the namespace in BEB.                                                        |
-| EVENT_TYPE_PREFIX       |               | The prefix of the eventType as per the BEB event specification.                                    |
-
+| Environment Variable    | Default Value | Description                                                                                |
+| ----------------------- | ------------- |------------------------------------------------------------------------------------------- |
+| INGRESS_PORT            | 8080          | The ingress port for the CloudEvents Gateway Proxy.                                        |
+| MAX_IDLE_CONNS          | 100           | The maximum number of idle (keep-alive) connections across all hosts. Zero means no limit. |
+| MAX_IDLE_CONNS_PER_HOST | 2             | The maximum idle (keep-alive) connections to keep per-host. Zero means the default value.  |
+| REQUEST_TIMEOUT         | 5s            | The timeout for the outgoing requests to the Messaging server.                             |
+| CLIENT_ID               |               | The Client ID used to acquire Access Tokens from the Authentication server.                |
+| CLIENT_SECRET           |               | The Client Secret used to acquire Access Tokens from the Authentication server.            |
+| TOKEN_ENDPOINT          |               | The Authentication Server Endpoint to provide Access Tokens.                               |
+| EMS_PUBLISH_URL         |               | The Messaging Server Endpoint that accepts publishing CloudEvents to it.                   |
+| BEB_NAMESPACE           |               | The name of the namespace in BEB.                                                          |
+| EVENT_TYPE_PREFIX       |               | The prefix of the eventType as per the BEB event specification.                            |
 
 ## Flags
-| Flag | Default Value | Description                                                                                   |
-| ----------------------- | ------------- |---------------------------------------------------------------------------------------------- |
-| maxRequestSize | 65536 | The maximum size of the request. |
+| Flag                    | Default Value | Description                                                                                |
+| ----------------------- | ------------- |------------------------------------------------------------------------------------------- |
+| max-request-Size        | 65536         | The maximum size of the request.                                                           |
```

Comment:
here should be "max-request-size", right?

Reply:
Done